packages/venia-concept/src/components/MiniCart/product.js

```diff
@@ -51,28 +48,35 @@ class Product extends Component {
 
     render() {
         const { options, props } = this;
-        const { classes, item } = props;
-        const amount = item.price.regularPrice.amount;
+        const { classes, item, currencyCode } = props;
 
         return (
             <li className={classes.root}>
-                <div className={classes.image} />
+                <div
+                    className={classes.image}
+                    style={{
+                        height: imageHeight,
+                        width: imageWidth,
+                        backgroundImage: `url(${makeProductMediaPath(
+                            item.image.file
+                        )})`
+                    }}
+                />
                 <div className={classes.name}>{item.name}</div>
-                <dl className={classes.options}>{options}</dl>
+                {options.length > 0 ? (
+                    <dl className={classes.options}>{options}</dl>
+                ) : null}
```

Comment:
Consider [using a getter](https://github.com/magento-research/pwa-studio/blob/master/packages/venia-concept/src/components/ProductOptions/option.js#L25-L27) or a helper method, then [inject the result directly](https://github.com/magento-research/pwa-studio/blob/master/packages/venia-concept/src/components/MiniCart/miniCart.js#L82).